stable/concourse/templates/web-svc.yaml

```diff
@@ -7,12 +7,19 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  annotations:
+  {{- range $key, $value := .Values.web.service.annotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
 spec:
   type: {{ .Values.web.service.type }}
   ports:
     - name: atc
       port: {{ .Values.concourse.atcPort }}
       targetPort: atc
+    - name: atc-https
+      port: 443
+      targetPort: atc
     - name: tsa
       port: {{ .Values.concourse.tsaPort }}
       targetPort: tsa
```

Comment:
Can you also make nodePort configurable (if service type is LoadBalancer or NodePort)?